interpreter/cling/lib/Interpreter/NullDerefProtectionTransformer.cpp

```diff
@@ -213,18 +213,18 @@ namespace cling {
       Expr* VoidExprArg = utils::Synthesize::CStyleCastPtrExpr(&m_Sema,Context.VoidPtrTy,
                                                                (uint64_t)Arg);
 
-      Expr *args[] = {VoidSemaArg, VoidExprArg};
+      Expr *args[] = {VoidSemaArg, VoidExprArg, Arg};
 
       Scope* S = m_Sema.getScopeForContext(m_Sema.CurContext);
       DeclarationName Name
-        = &Context.Idents.get("cling__runtime__internal__throwNullDerefException");
+        = &Context.Idents.get("cling__runtime__internal__check__and__throw__if__invalid");
 
       SourceLocation noLoc;
       LookupResult R(m_Sema, Name, noLoc, Sema::LookupOrdinaryName,
                    Sema::ForRedeclaration);
       m_Sema.LookupQualifiedName(R, Context.getTranslationUnitDecl());
```

Comment:
That lookup shouldn't change - once we found it we found it. Can you store it as a member, such that we only need to do the lookup once?